src/components/include/cos_asm_simple_stacks.h

```diff
@@ -1,13 +1,17 @@
 #ifndef COS_ASM_SIMPLE_STACKS_H
 #define COS_ASM_SIMPLE_STACKS_H
 
+#ifndef MAX_STACK_SZ_BYTE_ORDER
+#error "Missing MAX_STACK_SZ_BYTE_ORDER, try including consts.h"
```

Comment:
Generally, this is a good way to check errors (a static assertion).  However, in this case, won't this just result a compile-time error if it is not defined?  So you add this, it gives a compile-time error; you don't, and it does the same?  If that's the case, I'd always err on less code = better.

I'm going to likely apply this anyway, since you're rushed, but another PR to change this (if you agree) would be good.